composer.json

```diff
@@ -61,14 +61,14 @@
         "illuminate/validation": "^6.0",
         "illuminate/view": "^6.0",
         "intervention/image": "^2.5.0",
-        "laminas/laminas-diactoros": "^1.8.4",
-        "laminas/laminas-httphandlerrunner": "^1.0",
-        "laminas/laminas-stratigility": "^3.0",
+        "laminas/laminas-diactoros": "^2.4.1",
+        "laminas/laminas-httphandlerrunner": "^1.2.0 || ^1.3.0",
```

Comment:
Didn't we change these sometime previously in the PR's history?

Reply:
Right.